deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/layers/convolutional/KerasSeparableConvolution2D.java

```diff
@@ -72,9 +72,11 @@ public KerasSeparableConvolution2D(Map<String, Object> layerConfig, boolean enfo
         super(layerConfig, enforceTrainingConfig);
 
         hasBias = getHasBiasFromConfig(layerConfig, conf);
-        numTrainableParams = hasBias ? 2 : 1;
+        numTrainableParams = hasBias ? 3 : 2;
```

Comment:
For my own understanding, can you explain what happened here?

Reply:
Ah I think it's because we added depth multiplier.

Reply:
no, it was just wrong before and I didn't notice. sep conv has two sets of weights (depth-wise and point-wise) plus optionally bias.